Pipfile

```
@@ -4,6 +4,10 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+app = "*"
+flask = "*"
+kafka-python = "*"
+"s3fs" = "*"
 
```

`gunicorn` is missing here
```suggestion
"gunicorn" = "*"

```